src/main/java/org/shaolinmasters/akkadianlexicon/models/WebContents.java

```diff
@@ -0,0 +1,27 @@
+package org.shaolinmasters.akkadianlexicon.models;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Table(name = "web_contents")
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class WebContents {
+
+  @Id
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  private Long id;
+  @Lob
+  @Column(nullable = false, columnDefinition = "varchar(20)")
+  private String content;
```

Comment:
Itt irtad is, hogy nem kell a varchar felett @Lob annotacio, igazad van. Annyi csak, hogy sztem felcserelted a valtozokat. A content valtozot akartad MEDIUMTEXT-nek, hiszen ez fogja tartalmazni az adatot es a title valtozot akartad varcharnak.
Sztem amugy nem fontos limitalni a varchar meretet, maradhat a default, uh en azt a columnDefinition-t szivem szerint elhagynam, de ez mar apro reszlet, akar benne is hagyhatod.